pkg/core/transaction/transaction.go

```diff
@@ -72,6 +78,11 @@ type Transaction struct {
 	// Trimmed indicates this is a transaction from trimmed
 	// data.
 	Trimmed bool
+
+	// NetworkFee of the transaction divided by its size.
+	// This field is being cached and should be retrieved via
+	// FeePerByte() method.
+	feePerByte util.Fixed8
```

Comment:
Why do we need it in the `Transaction`?

Reply:
Removed this field and leave only `FeePerByte()` method.